data_pipeline/models/github/pull_requests/pull_requests.sql

```diff
@@ -52,6 +52,7 @@ final as (
       id,
       -- replace useless internal PR ID with something meaningful
       repos.repo_name || '/' || p.pull_request_number as pull_request_id,
+      {{ extract_org_name("p.pull_request_url") }} AS pr_org_name,
```

Comment:
You have to generate one more column and make it the new reference to users table, which no longer has user_id as the primary key.